packages/core/test/integration/sequelize/truncate.test.ts

```diff
@@ -86,4 +86,58 @@ describe('Sequelize#truncate', () => {
       expect(await B.count()).to.eq(0);
     });
   }
+
+  if (sequelize.dialect.supports.constraints.deferrable) {
```

Comment:
This is the same test as above but the above test used deferred constraints because an earlier version of the test had non-nullable foreign keys

So I think these two new tests can be removed

Reply:
Ok, I've removed these now